src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -301,9 +342,46 @@ def _process_genome_attri_tools(genome_attr_tools: set[str],
         docs.extend(parse_ops(root_dir, kbase_collection, load_ver))
 
     docs = merge_docs(docs, '_key')
+    docs = _add_upa_info(docs, root_dir, kbase_collection, load_ver, tool)
     output = f'{kbase_collection}_{load_ver}_{"_".join(genome_attr_tools)}_{COMPUTED_GENOME_ATTR_FILE_SUFFIX}'
     _create_import_files(root_dir, output, docs)
 
+    export_types_output = f'{kbase_collection}_{load_ver}_{"_".join(ENCOUNTERED_TYPES)}_{KBCALL_EXPORT_TYPES_FILE_SUFFIX}'
```

Comment:
I don't think you need the encountered types in the doc name. Just the collection and load ver should make it unique

Reply:
Updated!

Reply:
:+1: